packages/dart/npt_flutter/lib/features/onboarding/util/activate_util.dart

```diff
@@ -9,6 +9,8 @@ import 'package:at_onboarding_flutter/src/utils/at_onboarding_response_status.da
 import 'package:at_server_status/at_server_status.dart';
 import 'package:http/http.dart';
 import 'package:http/io_client.dart';
+// ignore: implementation_imports
+import 'package:at_onboarding_flutter/src/utils/at_onboarding_app_constants.dart';
```

Comment:
This is exposed so we can set the root domain in one of the places in at_onboarding_flutter